src/Lib.hs

```diff
@@ -1,8 +1,8 @@
 module Lib
-    ( someFunc
+    ( palette256
     ) where
 
-import Rainbow (putChunkLn, chunk, fore, blue)
+import Rainbow (putChunk, chunk, (&), back, color256)
 
-someFunc :: IO ()
-someFunc = putChunkLn $ fore blue $ chunk "Some blue text"
+palette256 :: IO ()
+palette256 =  mapM_ putChunk $ map ((chunk " " &) . back . color256) [0, 1..255]
```

Comment:
Why not just include `0` in range generation?

Reply:
Agree, I thought that it's need to set step